ccloud/docs/beginner-cloud.rst

```diff
@@ -0,0 +1,902 @@
+   command:
+
+   .. code-block:: bash
+
+       ccloud environment use env-5qz2q
+
+   You should see the following message: ``Now using "env-5qz2q" as the default (active) environment.``
+
+
+Create a new Kafka cluster and specify it as the default
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Run the following command to create a new |ccloud| cluster
+   ``demo-kafka-cluster``:
+
+   .. code-block:: bash
+
+      ccloud kafka cluster create demo-kafka-cluster --cloud aws --region us-west-2
+
+   .. note::
+
+      It may take up to 5 minutes for the |ak| cluster to be ready.
+
+#. Verify you see the following output:
+
+   .. code-block:: text
+
+      +--------------+---------------------------------------------------------+
+      | Id           | lkc-x6m01                                               |
+      | Name         | demo-kafka-cluster                                      |
+      | Type         | BASIC                                                   |
+      | Ingress      |                                                     100 |
+      | Egress       |                                                     100 |
+      | Storage      |                                                    5000 |
+      | Provider     | aws                                                     |
+      | Availability | LOW                                                     |
+      | Region       | us-west-2                                               |
+      | Status       | UP                                                      |
+      | Endpoint     | SASL_SSL://pkc-4kgmg.us-west-2.aws.confluent.cloud:9092 |
+      | ApiEndpoint  | https://pkac-ldgj1.us-west-2.aws.confluent.cloud        |
+      +--------------+---------------------------------------------------------+
+
+#. Specify ``lkc-x6m01`` as the active |ak|| cluster by running the following
+   command:
+
+   .. code-block:: bash
+
+      ccloud kafka cluster use lkc-x6m01
+
+   Running the previous command sets the |ak| cluster ``lkc-x6m01`` as the active
+   cluster for environment ``env-5qz2``.
+
+
+Create a user key/secret pair and specify it as the default
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Run the following command to create a user key/secret pair:
+
+   .. code-block:: bash
+
+      cloud api-key create --description "Demo credentials" --resource lkc-x6m01 -o json
+
+   The output is in JSON format:
+
+   .. code-block:: text
+
+      {
+         "key": "QX7X4VA4DFJTTOIA",
+         "secret": "fjcDDyr0Nm84zZr77ku/AQqCKQOOmb35Ql68HQnb60VuU+xLKiu/n2UNQ0WYXp/D"
+      }
+
+#. Associate the API key ``QX7X4VA4DFJTTOIA`` to the Kafka cluster ``lkc-x6m01``:
+
+   .. code-block:: bash
+
+      ccloud api-key use QX7X4VA4DFJTTOIA --resource lkc-x6m01
+
+   Running the previoud command sets the API Key "QX7X4VA4DFJTTOIA" as the
+   active API key for ``lkc-x6m0``.
+
+#. Verify you see the following message:
```

Comment:
Actually this is from the script, not from the command.  I think we need to remove this but tell the user something like "Please wait 90 seconds for Confluent Cloud cluster to be ready and for credentials to propagate"

Reply:
Got it